libtransmission/bandwidth.h

```diff
@@ -97,140 +99,160 @@ struct tr_band
  *   you'll only need to invoke it for the top-level tr_session bandwidth.
  *
  *   The peer-ios all have a pointer to their associated tr_bandwidth object,
- *   and call tr_bandwidthClamp() before performing I/O to see how much
+ *   and call tr_bandwidth::clamp() before performing I/O to see how much
  *   bandwidth they can safely use.
  */
 struct tr_bandwidth
 {
     /* these are PRIVATE IMPLEMENTATION details that should not be touched.
      * it's included in the header for inlining and composition. */
-
-    struct tr_band band[2];
+private:
```

Comment:
This should be reorganized to follow https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-order e.g.

> Use the `public` before `protected` before `private` order.

Reply:
Moved fields down after the private functions. Removed duplicate `private:` access tag.